webpack.config.js

```diff
@@ -12,16 +12,21 @@ function getEntry() {
   const mod = JSON.parse(npmListRes);
   const unbundledModule = ['impor', 'uuid',
```

Comment:
For reviewers: This entire webpack configuration seems a little wild to me. It appears to recursively find every single module that the main module depends on, then declare them all as entry points for bundling. I can't imagine the extension is actually doing that many dynamic imports, but in the interest of reducing the risk of this particular PR I tried to keep my changes a minimal as possible to get the build to work.

Reply:
I agree, I think we could try and clean up this file at some point, but this PR isn't the place to do it.